### PR TITLE
Execute E2E tests in a single electron/webdriver run

### PR DESCRIPTION
**Pull request by sytolk**

### CODE REVIEW

File: app/components/dialogs/settings/SettingsDialog.tsx [hunk 1]
```diff
@@ -227,6 +231,28 @@ const SettingsDialog = (props: Props) => {
...
+          onClose={() => {
+            setIsResetSettingsDialogOpened(false);
+          }}
+          title="Confirm"
+          content={i18n.t('core:confirmResetSettings')}
+          confirmCallback={result => {
+            if (result) {
+              const electron = window.require('electron');
+              const webContents = electron.remote.getCurrentWebContents();
+              webContents.session.clearStorageData();
```
Comment (uggrock) [inline app/components/dialogs/settings/SettingsDialog.tsx:246]: why not localStorage.clear(); https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear

File: app/components/dialogs/settings/SettingsDialog.tsx [hunk 2]
```diff
@@ -227,6 +231,28 @@ const SettingsDialog = (props: Props) => {
...
+              webContents.reload();
```
Comment (uggrock) [inline app/components/dialogs/settings/SettingsDialog.tsx:247]: this should work also on Cordova and Web versions

Comment (sytolk) [reply app/components/dialogs/settings/SettingsDialog.tsx:247]: Yes resetSettings button is displayed only in Electron

File: app/components/dialogs/settings/SettingsDialog.tsx (hunk 1 above)
Comment (sytolk) [reply app/components/dialogs/settings/SettingsDialog.tsx:246]: localStorage will be undefined in Electron app

**Pull request merged**